dev-libs/hyperscan/hyperscan-5.0.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="High-performance regular expression matching library"
+SRC_URI="https://github.com/01org/hyperscan/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://01.org/hyperscan"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_ssse3 static-libs"
+
+DEPEND="dev-util/ragel
+	=dev-lang/python-2*
```

Comment:
Depending on `dev-lang/python` directly is unacceptable. Use appropriate eclass instead (read docs on https://wiki.gentoo.org/wiki/Project:Python).